Tasks/AzureWebPowerShellDeployment/task.json

```diff
@@ -71,7 +71,7 @@
     },
     {
       "name": "AdditionalArguments",
-      "type": "string",
+      "type": "multiLine",
```

Comment:
You would now want to deal with newline in input before you pass the arguments to Azure command-let.